### ISSUE TITLE
test: TestCtlV3MemberRemove

### ISSUE THREAD

**Issue by xiang90** [coreos/etcd#7204]
```
--- FAIL: TestCtlV3MemberRemove (4.29s)
	ctl_v3_member_test.go:91: read /dev/ptmx: input/output error (expected "9366cce900fbd794 removed from cluster 4529f4d7b68cdaf9", got ["Error:  etcdserver: server stopped\r\n"])
```

**Comment by xiang90** [comment]
this is a dup with #7187.

**Comment by heyitsanthony** [comment]
This was not fixed by #7242
